community/logging/src/test/java/org/neo4j/logging/RotatingFileOutputStreamSupplierTest.java

```diff
@@ -235,9 +238,15 @@ public void rotationCompleted( OutputStream out )
         } );
 
         executor.shutdown();
+        boolean terminated = executor.awaitTermination( 20, TimeUnit.SECONDS );
```

Comment:
This timeout is longer than the timeout for the test. That seems strange, is it intentional?